src/main/java/ro/vladutit/Don/t/forget/v2/repository/UserRepository.java

```diff
@@ -1,10 +1,13 @@
 package ro.vladutit.Don.t.forget.v2.repository;
 
+import org.springframework.data.jpa.repository.Query;
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 import ro.vladutit.Don.t.forget.v2.model.User;
 
 @Repository
 public interface UserRepository extends CrudRepository<User, Long> {
+
+    @Query("SELECT u FROM User u WHERE u.email = ?1")
```

Comment:
why do you have sql code here??

Reply:
I read these days about Queries on the Spring documentation and from what I understand, it's that it works very well without this line for a small number of queries, but I think that in the future, when the application grows, this line will help . I read this here [LINK](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#jpa.query-methods.at-query). What do you think?

Reply:
ok then, good catch👍🏻

Reply:
Thank you!